src/stories/atoms/Header/Header.tsx

```diff
@@ -16,6 +17,8 @@ interface Props {
   subheadingText?: string | null;
   /** What size should the header be? */
   size?: SizesEnum;
+  /** What size would the screen be? */
+  screenSize: string;
```

Comment:
What's the difference between the `size` prop and the `screenSize` prop? Could we use `screenSize` in the parent or calling component to determine what the `size` prop should be and save ourselves from passing another prop?

Reply:
Right. They do serve similiar purposes. In stead of having both screenSize and size I could just keep size and pass it in <Wrapper /> as the only prop. This way the CSS setting for <Wrapper /> will base on the comparison between size prop and SizesEnum.Small (or Medium or Large, depending on what we want to achieve).